app/policies/comment_policy.rb

```diff
@@ -0,0 +1,16 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def create?
+    user.present?
```

Comment:
`user.present?` is always true since we enforce `authenticate_user` in application controller. Could just write `true`